Packs/Threatgraph/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Threatgraph",
+    "description": "ThreatGraph : this is created from scratch to address the issue of threatgraph being deprecated by CS but there being no replacement",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Neil Walker",
+    "url": "",
+    "email": "",
+    "created": "2022-05-24T09:54:38Z",
+    "categories": [],
```

Comment:
Please supply at least one category, for example: 'Analytics & SIEM', 'Utilities', 'Messaging', 'Endpoint', 'Network Security', 'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis', 'IT Services', 'Data Enrichment & Threat Intelligence', 'Authentication', 'Database', 'Deception', 'Email Gateway', 'Identity and Access Management', 'File Integrity Management'

Reply:
done in last pull request